src/components/global/BaseSelect.vue

```diff
@@ -10,71 +10,127 @@ export default defineComponent({
   },
   props: {
     selectOptions: {
-      type: Array,
-      default() {
-        return [];
-      }
```

Comment:
Why the removal of the default? 🤔

Reply:
I was tinkering with a different approach of displaying the `currentOption` which the default value was somehow breaking and forgot to change it.. Should have left a comment for myself!

Reply:
I'd definitely try to leave the default value though for the array of options 😄

Reply:
Yes - totally - in this solution it doesn't break anything, I'll change it immediately!

Reply:
Done :smile: